### PR TITLE
sys var: add global scope for `tidb_low_resolution_tso`

### PR DESCRIPTION
**Pull request by cfzjywxk**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Related to the tidb issue https://github.com/pingcap/tidb/issues/54999, expaning the system variable scope to `GLOBAL` for the `tidb_low_resolution_tso`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3003,12 +3003,12 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 ### `tidb_low_resolution_tso`
 
-- 作用域：SESSION
+- 作用域：SESSION | GLOBAL
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
 - 默认值：`OFF`
 - 这个变量用来设置是否启用低精度 TSO 特性。开启该功能之后，TiDB 使用缓存 Timestamp 来读取数据。缓存 Timestamp 默认每 2 秒更新一次。从 v8.0.0 开始，你可以通过 [`tidb_low_resolution_tso_update_interval`](#tidb_low_resolution_tso_update_interval-从-v800-版本开始引入) 配置缓存 Timestamp 的更新时间间隔。
-- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。
+- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。自 v8.3.0 版本开始，该变量可设置 GLOBAL scope。
```
Comment (hfxsd) [inline system-variables.md:3011]:
```suggestion
- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。自 v8.3.0 版本开始，该变量支持 GLOBAL 作用域。
```

File: system-variables.md [hunk 2]
```diff
@@ -3003,12 +3003,12 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 ### `tidb_low_resolution_tso`
 
-- 作用域：SESSION
+- 作用域：SESSION | GLOBAL
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
 - 默认值：`OFF`
 - 这个变量用来设置是否启用低精度 TSO 特性。开启该功能之后，TiDB 使用缓存 Timestamp 来读取数据。缓存 Timestamp 默认每 2 秒更新一次。从 v8.0.0 开始，你可以通过 [`tidb_low_resolution_tso_update_interval`](#tidb_low_resolution_tso_update_interval-从-v800-版本开始引入) 配置缓存 Timestamp 的更新时间间隔。
-- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。
+- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。自 v8.3.0 版本开始，该变量支持 GLOBAL 作用域。
```
Comment (hfxsd) [inline system-variables.md:3011]:
```suggestion
- 主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 TSO 的开销。
- 从 v8.3.0 版本开始，该变量支持 GLOBAL 作用域。
```

**Review by joey-yez — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**